docs/versioned-plugins/filters/ruby-v3.1.0.asciidoc

```diff
@@ -124,7 +124,7 @@ end
 
 We can now test that the ruby script we're using is implemented correctly:
 
-[source]
+[source,shell]
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin. We can fix it with a simple search/replace in the old files.